### DISCUSSION TITLE
Supabase Auth as SSO Provider

### DISCUSSION THREAD

**Discussion by dmudro** [supabase/supabase#10730] [Questions]
how would I go about using Supabase as indentity provider in another (3rd party) service in an SSO fashion?

Supabase obviously supports many providers though this would be a reverse situation where I want to use my Supabase Auth as identity source / provider, ideally OAuth or SAML.

imagine a third party SaaS in which I want my users to be able to "sign in with your ACME account", where ACME is my Supabase powered app.

Auth0 take: https://auth0.com/docs/authenticate/single-sign-on/inbound-single-sign-on

**Comment by nileshtrivedi** [comment]
💯 for this feature request. It's a very frequent need to delegate auth of multiple services to the primary supabase app.

I did some digging. [According to the docs](https://supabase.com/docs/guides/self-hosting#architecture), Supabase's auth APIs are based on [GoTrue](https://github.com/supabase/gotrue) which [inherited a config parameter](https://github.com/supabase/gotrue#inherited-features) called `GOTRUE_SAML_ENABLED` from Netlify's original codebase but it is explicitly not supported by Supabase. But the [code seems to be present](https://github.com/supabase/gotrue/blob/5caacc1f81ff20f8f09f4598b510c767e589a1c5/internal/api/api.go#L171).

It might be possible to get this working in a self-hosted Supabase setup.

**Reply by nileshtrivedi** [reply]
A more practical path here might be to run a dedicated SAML provider like [Authentic](https://goauthentik.io/) or Auth0 and delegate both your supabase app and your new app's authentication to this via SAML. But in this case, the user registration UX might not be ideal.
